sdk/src/wallet/account/update.rs

```diff
@@ -22,7 +21,7 @@ use crate::{
     },
 };
 
-impl AccountHandle {
+impl Account {
     /// Set the alias for the account
     pub async fn set_alias(&self, alias: &str) -> crate::wallet::Result<()> {
         let mut account = self.write().await;
```

Comment:
Want to use details here? Could also stay, not important

Reply:
I think there are plenty of places where this could be changed but it's not very easy to search for them, I'll see what I can do